### DISCUSSION TITLE
AddChatGptCore method

### DISCUSSION THREAD

**Discussion by N1K0232** [marcominerva/ChatGptNet#58] [General]
I have a question about the AddChatGptCore extension method... It's not a bad practice to assign directly and clearly the base url of the client?
Usually it's not better to read it from an internal file with our constants?

**Comment by marcominerva** [comment]
Actually, I was thinking about adding a property to `ChatGptOptions` to set base service address  (using the current value as default).

**Reply by N1K0232** [reply]
Oh okay. Thank you so much. And sorry for the mess in the welcome message. I haven't noticed

**Reply by marcominerva** [reply]
No problem, thank you for raising the questions 😀
